### PR TITLE
Adding comments action on pull requests

### PR DESCRIPTION
**Pull request by tomasbjerre**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -5,6 +5,7 @@ bin
 out 
 src/generated
 .gradle
+test-output
```
Comment (cdancy) [inline .gitignore]: What is this for?

**Comment by tomasbjerre** [comment]
I think I got it when running TestNG from Eclipse.

File: .gitignore (hunk 1 above)
Comment (tomasbjerre) [inline .gitignore]: I think I got it when running TestNG from Eclipse.

**Comment by cdancy** [comment]
Bitbucket documentation is always either hit or miss. This is one of those cases [where it's a miss IMO.](https://developer.atlassian.com/static/rest/bitbucket-server/latest/bitbucket-rest.html#idm45888278617264). Can `path` be null? What is it suppose to represent?

**Comment by tomasbjerre** [comment]
Right, yes it should be:
https://developer.atlassian.com/static/rest/bitbucket-server/latest/bitbucket-rest.html#idm45888278617264

Path cannot be null.

I forgot to add POST, can I add that to this PR?

**Comment by cdancy** [comment]
@tomasbjerre I'll take this one but lets put `POST` functionality in a separate PR. 

Are you able to run live tests? If so, and as part of the subsequent `POST` PullRequest, can you write 2 live tests: one for posting a comment and one for listing all comments? I recognize that I still have not yet made it stupid-simple to run live tests against a system so feel free to change the parameters to point at something more local to you to ensure all works as expected.

**Comment by tomasbjerre** [comment]
I just noticed that POST [is actually implemented](https://github.com/cdancy/bitbucket-rest/blob/master/src/main/java/com/cdancy/bitbucket/rest/features/CommentsApi.java#L67)! So wont be doing that.

**Comment by cdancy** [comment]
Forgot about that ;) [I actually use the more simpler version](https://github.com/cdancy/bitbucket-rest/blob/master/src/main/java/com/cdancy/bitbucket/rest/features/CommentsApi.java#L56) which is probably why this slipped my mind.

**Pull request merged**